src/database/schemas/questionnaire/questionnaire.ts

```diff
@@ -0,0 +1,19 @@
+export interface QuestionnaireModel {
+  id: number;
+  title: string;
+  description: string;
+  options: Question[];
+  tags?: string[];
+  passedUsers?: number[];
+  passScore: number;
+}
+
+export interface Question {
+  question: string;
```

Comment:
Could this be question -> questionText, and in options value could be isCorrect ?
Do we need interface to passedUsers? Or could all that information be in one object ?
Could passScore be more like passingScore or minPassScore ?